src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,72 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0501: r##"
+This error indicates that a mutable variable which has been captured in a
+closure is being used while it is still captured by the closure. Because the
+closure has borrowed the variable, the variable is not available for use until
```

Comment:
The switch from the term "captured" to the term "borrowed" may be confusing for some. It may be useful to put in some note that the capture is a borrow, or to stick with one term (probably "borrowed").